### ISSUE TITLE
555.42.06 is available

### ISSUE THREAD

**Issue by TLCFEM** [flathub/org.freedesktop.Platform.GL.nvidia#265]
```bash
apt list --installed | grep nvidia-driver
nvidia-driver-bin/unknown,now 555.42.06-1 amd64 [installed,automatic]
nvidia-driver-libs/unknown,now 555.42.06-1 amd64 [installed,automatic]
nvidia-driver/unknown,now 555.42.06-1 amd64 [installed,automatic]
```

**Comment by EriksRemess** [comment]
both https://us.download.nvidia.com/tesla/555.42.06/NVIDIA-Linux-x86_64-555.42.06.run & https://us.download.nvidia.com/XFree86/Linux-x86_64/555.42.06/NVIDIA-Linux-x86_64-555.42.06.run return 404

**Comment by TLCFEM** [comment]
> both https://us.download.nvidia.com/tesla/555.42.06/NVIDIA-Linux-x86_64-555.42.06.run & https://us.download.nvidia.com/XFree86/Linux-x86_64/555.42.06/NVIDIA-Linux-x86_64-555.42.06.run return 404

Yes, seems the server has not been updated yet, but apt repo has been updated. Maybe need to wait for a few hours more.

**Comment by EriksRemess** [comment]
Made PR for cuda runfile. Tested locally and GreenWithEnvy picks up new version correctly 
![image](https://github.com/flathub/org.freedesktop.Platform.GL.nvidia/assets/176241/7b91264d-00ac-4625-9ccf-13f5fae8c044)

**Comment by EriksRemess** [comment]
PR updated with runfiles hosted on GitHub

**Comment by 404NetworkError** [comment]
Closed in #266.
